[BUG] Missing revision info in databricks shims failed nightly build

**Describe the bug**
related to https://github.com/NVIDIA/spark-rapids/pull/9148

databricks aggregators or new api artifacts now have no revision property files, 
which would fail the parallel worlds check in nightly `error=Error: version file missing`

```
[2023-09-10T11:43:46.379Z] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:3.1.0:run (create-parallel-world) on project rapids-4-spark_2.12: An Ant BuildException has occured: The following error occurred while executing this line:
[2023-09-10T11:43:46.379Z] [ERROR] /home/jenkins/agent/workspace/jenkins-rapids_nightly-dev-github-925/dist/maven-antrun/build-parallel-worlds.xml:113: check-shims-revisions.sh failed: exitCode=1 error=Error: version file missing: /home/jenkins/agent/workspace/jenkins-rapids_nightly-dev-github-925/dist/target/parallel-world/spark321db/rapids4spark-version-info.properties
[2023-09-10T11:43:46.379Z] [ERROR] around Ant part ...<ant antfile="/home/jenkins/agent/workspace/jenkins-rapids_nightly-dev-github-925/dist/maven-antrun/build-parallel-worlds.xml" target="remove-dependencies-from-pom" />... @ 9:172 in /home/jenkins/agent/workspace/jenkins-rapids_nightly-dev-github-925/dist/target/antrun/build-main.xml
[2023-09-10T11:43:46.379Z] [ERROR] -> [Help 1]
```

**Steps/Code to reproduce bug**
https://github.com/NVIDIA/spark-rapids/blob/branch-23.10/jenkins/spark-nightly-build.sh

**Expected behavior**
Pass the revision check,

**Environment details (please complete the following information)**
databricks

**Additional context**
Add any other context about the problem here.

--- a/jenkins/databricks/deploy.sh
+++ b/jenkins/databricks/deploy.sh
@@ -30,7 +30,7 @@ VERSION_NUM=${BASE_SPARK_VERSION_TO_INSTALL_DATABRICKS_JARS//.}
 SPARK_VERSION_STR=spark$VERSION_NUM
 SPARK_PLUGIN_JAR_VERSION=`mvn help:evaluate -q -pl dist -Dexpression=project.version -DforceStdout`
 DB_SHIM_NAME=${SPARK_VERSION_STR}db
-DBJARFPATH=./aggregator/target/${DB_SHIM_NAME}/rapids-4-spark-aggregator_$SCALA_VERSION-$SPARK_PLUGIN_JAR_VERSION.jar
+DBJARFPATH=./aggregator/target/${DB_SHIM_NAME}/rapids-4-spark-aggregator_$SCALA_VERSION-$SPARK_PLUGIN_JAR_VERSION-${DB_SHIM_NAME}.jar
 echo "Databricks jar is: $DBJARFPATH"
 MVN="mvn -Dmaven.wagon.http.retryHandler.count=3 -DretryFailedDeploymentCount=3"
 $MVN -B deploy:deploy-file $MVN_URM_MIRROR -Durl=$SERVER_URL -DrepositoryId=$SERVER_ID \

